src/components/Proposals/InfoBoxes.js

```diff
@@ -0,0 +1,187 @@
+  const { layoutName } = useLayout()
+  const compactMode = layoutName === 'small'
+
+  const totalVotes = parseFloat(vote.yeas) + parseFloat(vote.nays)
+  const support = safeDiv(parseFloat(vote.yeas), totalVotes)
+  const quorumProgress = safeDiv(vote.yeas, vote.votingPower)
+  return (
+    <div
+      css={`
+        margin-top: ${2 * GU}px;
+        display: grid;
+        grid-template-columns: ${compactMode ? '1fr' : '1fr 1fr 1fr'};
+        grid-template-rows: ${compactMode ? '1fr 1fr 1fr' : '1fr'};
+        grid-gap: ${2 * GU}px;
+      `}
+    >
+      <div>
+        <Box
+          heading="Status"
+          css={`
+            height: 100%;
+          `}
+          padding={(compactMode ? 2 : 3) * GU}
+        >
+          <Status vote={vote} />
+        </Box>
+      </div>
+      <div>
+        <Box
+          heading={
+            <>
+              Support %
+              <Help hint="What is Support?">
+                <strong>Support</strong> is the relative percentage of tokens
+                that are required to vote “Yes” for a proposal to be approved.
+                For example, if “Support” is set to 50%, then more than 50% of
+                the tokens used to vote on a proposal must vote “Yes” for it to
+                pass.
+              </Help>
+            </>
+          }
+          css={`
+            height: 100%;
+          `}
+          padding={(compactMode ? 2 : 3) * GU}
+        >
+          <SummaryWithPercentages
+            size={support}
+            requiredSize={parseFloat(setting.formattedSupportRequiredPct)}
+          />
+        </Box>
+      </div>
+      <div>
+        <Box
+          heading={
+            <>
+              Minimum Approval %
+              <Help hint="What is Minimum Approval?">
+                <strong>Minimum Approval</strong> is the percentage of the total
+                token supply that is required to vote “Yes” on a proposal before
+                it can be approved. For example, if the “Minimum Approval” is
+                set to 20%, then more than 20% of the outstanding token supply
+                must vote “Yes” on a proposal for it to pass.
+              </Help>
+            </>
+          }
+          css={`
+            height: 100%;
+          `}
+          padding={(compactMode ? 2 : 3) * GU}
+        >
+          <SummaryWithPercentages
+            size={quorumProgress}
+            requiredSize={parseFloat(
+              setting.formattedMinimumAcceptanceQuorumPct
+            )}
+          />
+        </Box>
+      </div>
+    </div>
+  )
+}
+
+function SummaryWithPercentages({ size, requiredSize }) {
+  const theme = useTheme()
+
+  return (
+    <>
+      <div
+        css={`
+          ${textStyle('body2')};
+        `}
+      >
+        {round(size * 100, 2)}%{' '}
+        <span
+          css={`
+            color: ${theme.surfaceContentSecondary};
+          `}
+        >
+          (&gt;{requiredSize}% needed)
```

Comment:
Not opposed to using the encoded version, but we should generally be OK with using just `>` here to let React do its own manipulation :).